src/httpclient.c

```diff
@@ -588,8 +588,8 @@ int main(int argc, char **argv)
         case 'x':
             req.connect_to = h2o_mem_alloc(sizeof(*req.connect_to));
             if (h2o_url_init(req.connect_to, NULL, h2o_iovec_init(optarg, strlen(optarg)), h2o_iovec_init(NULL, 0)) != 0 ||
-                req.connect_to->_port == 0 || req.connect_to->_port == 65535) {
-                fprintf(stderr, "invalid server address specified for -X\n");
+                    req.connect_to->_port == 0 || req.connect_to->_port == 65535) {
```

Comment:
sorry for a nitpick but was this added by accident?

I'd strongly encourage [setting up `clang-format` and apply it](https://github.com/h2o/h2o/wiki/Coding-Style#formatting), if not already.

Reply:
Thanks for pointing it out!  I checked the coding guidelines and didn't see anything regarding multi-line conditional statement.  I usually indent them extra, but I have corrected it in [612cbbc5](https://github.com/h2o/h2o/pull/2691/commits/612cbbc5d6f55d6cb08ac2b90b33476ff97faeeb) using `clang-format`.